lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.spec.ts

```diff
@@ -0,0 +1,215 @@
+    template: ``
+})
+export class MockSearchDateRangeAdvancedComponent {
+    @Input()
+    dateFormat: string;
+    @Input()
+    maxDate: string;
+    @Input()
+    field: string;
+    @Input()
+    initialValue: SearchDateRangeAdvanced;
+
+    @Output()
+    changed = new EventEmitter<Partial<SearchDateRangeAdvanced>>();
+    @Output()
+    valid = new EventEmitter<boolean>();
+}
+describe('SearchDateRangeAdvancedTabbedComponent', () => {
+    let component: SearchDateRangeAdvancedTabbedComponent;
+    let fixture: ComponentFixture<SearchDateRangeAdvancedTabbedComponent>;
+    let betweenMockData: SearchDateRangeAdvanced, inLastMockData: SearchDateRangeAdvanced, anyMockDate: SearchDateRangeAdvanced;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [SearchDateRangeAdvancedTabbedComponent, SearchFilterTabbedComponent, SearchDateRangeAdvancedComponent],
+            imports: [
+                TranslateModule.forRoot(),
+                ContentTestingModule
+            ],
+            providers: [
+                { provide: SearchFilterTabbedComponent, useClass: MockSearchFilterTabbedComponent },
+                { provide: SearchDateRangeAdvancedComponent, useClass: MockSearchDateRangeAdvancedComponent }
+            ]
+        });
+        fixture = TestBed.createComponent(SearchDateRangeAdvancedTabbedComponent);
+
+        component = fixture.componentInstance;
+        component.id = 'dateRangeAdvanced';
+        component.context = {
+            queryFragments: {
+                dateRangeAdvanced: ''
+            },
+            update: jasmine.createSpy('update')
+        } as any;
+        component.settings = {
+            hideDefaultAction: false,
+            dateFormat: 'dd-MMM-yy',
+            maxDate: 'today',
+            field: 'createdDate, modifiedDate',
+            displayedLabelsByField: {
+                'createdDate': 'Created Date',
+                'modifiedDate': 'Modified Date'
+            }
+        };
+        component.tabsValidity = {
+            'createdDate': true,
+            'modifiedDate': true
+        };
+
+        betweenMockData = {
+            dateRangeType: DateRangeType.BETWEEN,
+            inLastValueType: InLastDateType.DAYS,
+            inLastValue: undefined,
+            betweenStartDate: parse('05-Jun-23', 'dd-MMM-yy', new Date()),
+            betweenEndDate: parse('07-Jun-23', 'dd-MMM-yy', new Date())
+        }
+        inLastMockData = {
+            dateRangeType: DateRangeType.IN_LAST,
+            inLastValueType: InLastDateType.WEEKS,
+            inLastValue: '5',
+            betweenStartDate: undefined,
+            betweenEndDate: undefined
+        }
+        anyMockDate = {
+            dateRangeType: DateRangeType.ANY,
+            inLastValueType: InLastDateType.DAYS,
+            inLastValue: null,
+            betweenStartDate: null,
+            betweenEndDate: null
+        }
```

Comment:
missing semicolon

Reply:
These has been resolved in [this](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/commits/bdaa0e52fd3a31ba7c3802c6cdc6a27a733d6057) commit